OWML.ModHelper/ModCombination.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper
+{
+	public class ModCombination : IModCombination
```

Comment:
The name ModCombination sounds like a combination of mods to me. Maybe something more descriptive like KeyCombination, InputCombination, etc?

Reply:
maybe ModInputCombination? To keep consistency with OWML's naming

Reply:
Yeah, sounds good